react/src/components/summoner/Summoner.tsx

```diff
@@ -338,31 +338,47 @@ export function Summoner({route, region, store}: {route: any; region: string; st
             <div className="row">
               <div className={`${custom_max_width}`}>
                 <div className="row">
-                  <div className="col l6 m12">
-                    <MatchFilterForm onUpdate={matchFilterOnUpdate} />
-                  </div>
-                  <div className="col l6 m12">
-                    <div
-                      style={{
-                        display: 'inline-block',
-                        verticalAlign: 'top',
-                      }}
-                    >
-                      <RecentlyPlayedWith
-                        region={region}
-                        summoner={summoner}
-                        matches={matches || []}
-                        store={store}
-                      />
+                  <div className="col l4 m12">
+                      <div
+                        style={{
+                          display: 'inline-block',
+                          verticalAlign: 'top',
+                        }}
+                      >
+                        <RecentlyPlayedWith
+                          region={region}
+                          summoner={summoner}
+                          matches={matches || []}
+                          store={store}
+                        />
+                      </div>
```

Comment:
There's some strange indentation going on here.  342 has 4 space indent. 355 closing div is not aligned.